inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/client/StreamSourceClient.java

```diff
@@ -105,6 +105,26 @@ public boolean deleteSource(int id) {
         return response.getData();
     }
 
+    /**
+     * Stop data source information by id.
+     */
+    public boolean stopSource(int id) {
+        Preconditions.expectTrue(id > 0, "sourceId is illegal");
```

Comment:
sourceId is excepted bigger than 0 but got %s

Reply:
Done